TestFoundation/TestNSStream.swift

```diff
@@ -124,6 +132,107 @@ class TestNSStream : XCTestCase {
         XCTAssertEqual(Stream.Status.error, fileStream.streamStatus)
     }
     
+    /* This test is the success case for Streams that are in the blessed list with the
+     * dataGetBuffer as part of the call back e.g
+     * static const struct _CFStreamCallBacksV1 readDataCallBacks = {1, readDataCreate,readDataFinalize, readDataCopyDescription, readDataOpen, NULL, dataRead, dataGetBuffer, dataCanRead, NULL, NULL, NULL, NULL, NULL, NULL, readDataSchedule, NULL};
+     */
+    func test_InputStreamGetBufferSuccessFromBlessedList(){
+        let message: NSString = "Hello, playground"
+        let messageData: Data = message.data(using: String.Encoding.utf8.rawValue)!
+        let dataStream = InputStream(data: messageData)
+        XCTAssertEqual(Stream.Status.notOpen, dataStream.streamStatus)
+        dataStream.open()
+        XCTAssertEqual(Stream.Status.open, dataStream.streamStatus)
+        let buffer = UnsafeMutablePointer<UnsafeMutablePointer<UInt8>?>.allocate(capacity: 1)
```

Comment:
This might be ok, I just wanted to double check that you were using the latest stuff. There are a lot of changes to UnsafeMutablePointer API going on.

cc @atrick